.github/workflows/iroha2-dev-pr.yml

```diff
@@ -55,12 +58,24 @@ jobs:
           toolchain: stable
           override: true
           components: llvm-tools-preview
+      - name: Add components to nightly channel
+        run: rustup component add llvm-tools-preview --toolchain nightly-2022-04-20-x86_64-unknown-linux-gnu
       - uses: taiki-e/install-action@cargo-llvm-cov
       #
       - uses: Swatinem/rust-cache@v1
-      - name: Run tests and generate lcov coverage report
-        run: mold --run cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
-      - name: Upload to codecov
+      - name: Run tests, with ${{ matrix.feature_flag }}
+        run: |
+          mold --run cargo llvm-cov clean --workspace
```

Comment:
What are we cleaning?

Reply:
Build artifacts cached by `Swatinem/rust-cache` that might affect the coverage, I anticipated.
I think, however, I can remove this line because any cache related to changed code will be disposed

Reply:
After removing this line in 0caff41, CI failed due to:

![image](https://user-images.githubusercontent.com/49983831/183987931-0ea02637-aa94-4d3f-aa1c-c2832b3b9768.png)
https://github.com/hyperledger/iroha/runs/7772984325?check_suite_focus=true#step:8:311

Re-running didn't help. After reverting in 7a761c0, CI succeeded.
I don't know the truth of the correlation between the failure and the removal of `cargo llvm-cov clean`, but I'd keep this cleaning step

Reply:
No space left on device. If we're that close to overflowing, we need to do something about it. My guess would be to disable incremental compilation in the CI.